docs/content/en/docs/Coding Milestones/PoC2023q1/edge-syncer.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Edge Syncer"
+date: 2023-03-22
+weight: 4
+description: >
+---
+
+{{% pageinfo %}}
+Edge syncer is a new syncer working with mailbox workspace in edge-mc. 
+{{% /pageinfo %}}
+
+![edge-syncer drawio](./images/edge-syncer-overview.png)
+
+#### Registering Edge Syncer on a p-cluster
+
+Edge-syncer can be deployed on p-cluster easily by the following steps.
+1. Create SyncTarget and Location
+  - Mailbox controller creates mailbox workspace automatically. 
+  - Get mailbox workspace id
+2. Use command to register edge-syncer
+   ```console
+   kubectl ws <mb-ws name>
+   kcp workload edge-sync <EM Sync Target name> --image <EM Syncer Image> -o edge-syncer.yaml
+   ```
+3. Deploy edge-syncer on p-cluster.
+
+![edge-syncer boot](./images/edge-syncer-boot.png)
+
+#### Deploy workload objects from edge-mc to p-cluster
+
+To deploy resources to p-clusters, create the following in workload management workspace
+- workload objects
+  - Some objects are denatured if needed.
+  - Other objects are as it is
+- APIExport/API Schema corresponding to CRD
+  - TBD: Conversion from CRD to APIExport/APISchema could be automated by using MutatingAdmissionWebhook on workload management workspace.
+- EdgeSyncConfig (This is going to be replaced with EdgePlacement spec) 
+
+![edge-syncer deploy](./images/edge-syncer-deploy.png)
+
+After this, Edge-mc will put the following in the mailbox workspace.
+- Workload objects (both denatured one and not-denatured one)
+- API Binding (to API Export on the workload management workspace)
+- EdgeSyncConfig CR
+
+**TODO**: This is something we should clarify..e.g. which existing controller(s) in edge-mc will cover, or just create a new controller to handle uncovered one. 
```

Comment:
I think it is the placement translator's job to produce put the workload objects and syncer config into the mailbox workspaces.

Reply:
Thank you! I added the description.